integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml

```diff
@@ -94,9 +94,6 @@ spec:
           value: $(context.pipelineRun.name)
     - name: provision-rosa
       when:
-        - input: "$(tasks.test-metadata.results.test-event-type)"
```

Comment:
I think we can change it to :
        - input: "$(tasks.test-metadata.results.component-name)"
          operator: in
          values: ["release-service"]

Reply:
And then change the integration context to "pull_request". WDYT?

Reply:
changed... The problem is that will run after pull request...

Reply:
@jinqi7 Can you review?

Reply:
Yes. One task is missed to be changed.